edx_django_utils/monitoring/code_owner/middleware.py

```diff
@@ -17,7 +17,11 @@ class CodeOwnerMetricMiddleware:
     Django middleware object to set custom metrics for the owner of each view.
 
     Uses Django Setting CODE_OWNER_MAPPINGS as documented in get_code_owner_from_module.
-    Note: Use '*' as a configured module if you want a catch-all when there is no other match.
+    To configure a catch-all, use '*'.  Here is example yml with an important note::
+
+        CODE_OWNER_MAPPINGS:
+          team-that-owns-everything:
+          - '*'  # IMPORTANT: you must surround * with quotes in yml
```

Comment:
**Author Concern:** I wonder if using * is too dangerous since it could break things if quotes aren't used in yaml.  Would it be better to use something like `catch-all`, which hopefully won't ever be a module name, or is this fine?